### PR TITLE
SVGStyleElement.disabled + HTMLStyleElement.disabled

### PR DESCRIPTION
**Pull request by hamishwillee**
FF104 adds support for [`SVGStyleElement.disabled`](https://developer.mozilla.org/en-US/docs/Web/API/SVGStyleElement) (in https://bugzilla.mozilla.org/show_bug.cgi?id=1712623). 

This adds supporting docs. I'm going to do other properties in separate PR

### CODE REVIEW

File: files/en-us/web/api/htmlstyleelement/disabled/index.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+Note that there is no corresponding `disabled` attribute on the [HTML `<style>` element](/en-US/docs/Web/HTML/Element/style).
+
+## Value
+
+Returns `true` if the stylesheet is disabled, or there is no associated stylesheet; otherwise `false`.
+The value is `false` by default (if there is an associated stylesheet).
+
+The property can be used to enable or disable an associated stylesheet.
+Setting the property `true` when there is no associated stylesheet has no effect.
```
Comment (sideshowbarker) [inline files/en-us/web/api/htmlstyleelement/disabled/index.md:25]:
```suggestion
Setting the property to `true` when there is no associated stylesheet has no effect.
```

File: files/en-us/web/api/htmlstyleelement/disabled/index.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+## Examples
+
+### Disabling an inline style
+
+This example demonstrates programmatically setting the disabled property on a style that was defined in the HTML using the [HTML `<style>` element](/en-US/docs/Web/HTML/Element/style).
+Note that you can also access any/all stylesheets in the document using [`Document.styleSheets`](/en-US/docs/Web/API/Document/styleSheets).
+
+#### HTML
+
+The HTML contains a [HTML `<style>` element](/en-US/docs/Web/HTML/Element/style) element that makes paragraph elements blue, a paragraph element, and a button that will be used to enabled and disable the style.
```
Comment (sideshowbarker) [inline files/en-us/web/api/htmlstyleelement/disabled/index.md:36]:
```suggestion
The HTML contains an [HTML `<style>` element](/en-US/docs/Web/HTML/Element/style) element that makes paragraph elements blue, a paragraph element, and a button that will be used to enabled and disable the style.
```

File: files/en-us/web/api/svgstyleelement/disabled/index.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+Note that there is no corresponding `disabled` attribute on the [SVG `<style>` element](/en-US/docs/Web/SVG/Element/style).
+
+## Value
+
+Returns `true` if the stylesheet is disabled, or there is no associated stylesheet; otherwise `false`.
+The value is `false` by default (if there is an associated stylesheet).
+
+The property can be used to enable or disable an associated stylesheet.
+Setting the property `true` when there is no associated stylesheet has no effect.
```
Comment (sideshowbarker) [inline files/en-us/web/api/svgstyleelement/disabled/index.md:25]:
```suggestion
Setting the property to `true` when there is no associated stylesheet has no effect.
```

File: files/en-us/web/api/svgstyleelement/disabled/index.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+      stroke-width: 3px;
+    }
+  </style>
+  <circle cx="50" cy="50" r="25" />
+</svg>
+```
+
+#### JavaScript
+
+The code below gets the `style` element (a `SVGStyleElement`) using its id, and then sets it as disabled.
```
Comment (sideshowbarker) [inline files/en-us/web/api/svgstyleelement/disabled/index.md:53]:
```suggestion
The code below gets the `style` element (an `SVGStyleElement`) using its id, and then sets it as disabled.
```

**Review by sideshowbarker — APPROVED** [review]

File: files/en-us/web/api/svgstyleelement/disabled/index.md (hunk 4 above)
Comment (hamishwillee) [reply files/en-us/web/api/svgstyleelement/disabled/index.md]: Yoiks. Thanks for all of these.

**Pull request merged**